.github/workflows/revert.yml

```diff
@@ -46,5 +47,15 @@ jobs:
               python3 .github/scripts/trymerge.py --revert "${PR_NUM}"
             fi
           fi
+      - name: Comment on Canceled
+        if: ${{ cancelled() && steps.checkout.outcome == 'success' }}
+        continue-on-error: true
+        env:
+          GITHUB_TOKEN: ${{ secrets.MERGEBOT_TOKEN }}
+          PR_NUM: ${{ github.event.client_payload.pr_num }}
+          GH_RUN_URL: ${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}
+        run: |
+          set -ex
+          python3 .github/scripts/comment_on_pr.py "${PR_NUM}" "revert"
```

Comment:
Passing the PR_NUM twice via env variables and as a script parameter looks redundant to me. May be we should keep just one?

Reply:
We could just set it at the job level and not have to specify it twice

Reply:
moved it!